slack_sdk/oauth/redirect_uri_page_renderer/__init__.py

```diff
@@ -15,10 +15,22 @@ def __init__(
         self.success_url = success_url
         self.failure_url = failure_url
 
-    def render_success_page(self, app_id: str, team_id: Optional[str]) -> str:
+    def render_success_page(
+        self,
+        app_id: str,
+        team_id: Optional[str],
+        is_enterprise_install: Optional[bool],
+        enterprise_url: Optional[str],
```

Comment:
For backward-compatibility
```suggestion
        is_enterprise_install: Optional[bool] = None,
        enterprise_url: Optional[str] = None,
```